packages/plugin-core/src/utils/strings.ts

```diff
@@ -13,3 +13,8 @@ export const matchAll = (
 
   return out;
 };
+
+// https://stackoverflow.com/questions/18082/validate-decimal-numbers-in-javascript-isnumeric/1830844#1830844
+export const isNumeric = (n: any) => {
```

Comment:
can we add this in [common-all](https://github.com/dendronhq/dendron/blob/master/packages/common-all/src/utils.ts#L6:L6)? the goal is to embed most of the logic into the more foundational libraries so that non-vscode clients (eg. web) will eventually be able to call into them as well